### DISCUSSION TITLE
Bot does not react to any commands

### DISCUSSION THREAD

**Discussion by JuicyEFZ** [Raptor123471/DingoLingo#122] [Q&A]
I set the bot up and startet it.
I think the startup was good except one warning. 

![image](https://user-images.githubusercontent.com/79964083/207672673-5138c2fe-8f7a-4332-903f-6edc13a53e64.png)

I dont know if it has something to do with the load extension.

**Comment by TJ2005** [comment]
same issue

**Comment by techtechcy** [comment]
This warning is appearing because your Discord bot's message intents are not enabled. Message intents are a type of permission that allow your bot to view and respond to messages in Discord servers. To fix this, visit the Discord Developers Portal (discord.com/developers/applications), select your bot, navigate to the Bot tab, and enable Message Intents.
I hope this helps!
